packages/core/src/components/forms/_controls.scss

```diff
@@ -321,19 +329,23 @@ $control-indicator-spacing: $pt-grid-size !default;
   $switch-indicator-background-color: $white !default;
   $switch-indicator-background-color-disabled: rgba($white, 0.8) !default;
   $dark-switch-indicator-background-color: $dark-gray5 !default;
+  $dark-switch-indicator-box-shadow: 0 0 0 $button-border-width rgba($black, 0.8) !default;
```

Comment:
Explicitly set dark switch indicator's box shadow to not be inset, to be consistent with other indicator box shadows.